src/Raven.Server/Documents/BlittableMetadataModifier.cs

```diff
@@ -53,6 +55,14 @@ private DocumentFlags ReadFlags(JsonParserState state)
             return flags;
         }
 
+        private DateTime ReadDateTime(JsonParserState state)
+        {
+            var str = CreateLazyStringValueFromParserState(state);
+            if (DateTime.TryParseExact(str, DefaultFormat.DateTimeFormatsToRead, CultureInfo.InvariantCulture, DateTimeStyles.RoundtripKind, out DateTime time) == false)
```

Comment:
Maybe we should use LazyStringParser.TryParseDateTime?

Reply:
Done. Thanks.